db/migrate/20180412094012_change_money_field_type.rb

```diff
@@ -0,0 +1,7 @@
+class ChangeMoneyFieldType < ActiveRecord::Migration[5.1]
+  def change
+    change_column :campaigns, :current_amount, :float
```

Comment:
you do not need to have two migrations in your PR.
as I said you can delete them and have one.